dds2val/src/mapping.yml

```diff
@@ -0,0 +1,73 @@
+###############################################################################
+# Copyright (c) 2022 Robert Bosch GmbH
+#
+# All rights reserved. This configuration file is provided to you under the
+# terms and conditions of the Eclipse Distribution License v1.0 which
+# accompanies this distribution, and is available at
+# http://www.eclipse.org/org/documents/edl-v10.php
+#
+# SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
```

Comment:
This is the first time we use this SPDX-combination. Looking at https://www.eclipse.org/org/documents/edl-v10.php it only says BSD-3

Reply:
@SebastianSchildt what should we do. These are the licenses from the "real" author of this code. I did not change them.

Reply:
True, but I think it is fine. Question is always "should" you give configs a complex license, or maybe none at all? (that is why in "old EPL" times in KUKSA we used  EDL for stuff we did not deem "worthy" to be protected by EPL. 

The OR does not hurt here, question is, and it is acutally in line with the old dbc mapping: https://github.com/eclipse/kuksa.val.feeders/blob/v0.1.1/dbc2val/mapping.yml

However, to streamline this, I would suggest either removiong any license headers (as this is only config), but not sure how much license scanners like this
 or adding the normal apache 2.0 only (trusting people have enough criminal energy to "steal without attribution/given back" when needed)